src/lib/viewers/doc/DocFirstPreloader.js

```diff
@@ -218,8 +217,11 @@ class DocFirstPreloader extends EventEmitter {
     }
 
     getPreloadImageRequestPromises(preloadUrlWithAuth, pages, pagedPreLoadUrlWithAuth) {
-        const promise1 = this.api.get(preloadUrlWithAuth, { type: 'blob' });
-        const promises = [promise1];
+        const firstPageUrl = !pagedPreLoadUrlWithAuth
+            ? preloadUrlWithAuth
+            : pagedPreLoadUrlWithAuth.replace(PAGED_URL_TEMPLATE_PAGE_NUMBER_HOLDER, '1.webp');
+        const promise1 = this.api.get(firstPageUrl, { type: 'blob' });
+        const promises = [promise1.catch(e => e)];
```

Comment:
Where are these promises used?

Reply:
They are used in the showPreload function. They fetch the images